versioning.py

```diff
@@ -712,4 +723,3 @@ def commit(self):
         else:
             QMessageBox.information(self.iface.mainWindow(), "Info",
                     "There was no modification to commit")
```

Comment:
I think this should be replaced by a msg "infobar" (but not relevant for the current comit)